ERROR - 2023-02-11 08:04:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'smalan', 3306, '', 0)
#1 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\kodingan\inventaris\app\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Umum->index()
#7 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
#8 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\kodingan\inventaris\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 C:\xampp\htdocs\kodingan\inventaris\app\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
#5 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Umum->index()
#6 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
#7 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\kodingan\inventaris\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-02-11 08:04:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `piala`', [], false)
 3 SYSTEMPATH\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Umum->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
ERROR - 2023-02-11 08:04:40 --> Error connecting to the database: mysqli_sql_exception: Access denied for user 'root'@'localhost' (using password: YES) in C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:173
Stack trace:
#0 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(173): mysqli->real_connect('localhost', 'root', Object(SensitiveParameterValue), 'smalan', 3306, '', 0)
#1 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\kodingan\inventaris\app\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Umum->index()
#7 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
#8 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\kodingan\inventaris\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user '****'@'localhost' (using password: YES) in C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#3 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#4 C:\xampp\htdocs\kodingan\inventaris\app\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
#5 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Umum->index()
#6 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
#7 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\kodingan\inventaris\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 C:\xampp\htdocs\kodingan\inventaris\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#10 {main}
CRITICAL - 2023-02-11 08:04:40 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user '****'@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `piala`', [], false)
 3 SYSTEMPATH\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 4 APPPATH\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
 5 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Umum->index()
 6 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
 7 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
INFO - 2023-02-11 08:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:05:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:23:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:23:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:24:01 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:24:03 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:25:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:25:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 08:25:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-02-11 09:22:34 --> mysqli_sql_exception: Table 'inventaris_smalan.piala' doesn't exist in C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(295): mysqli->query('SELECT COUNT(*)...', 0)
#1 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('SELECT COUNT(*)...')
#2 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT COUNT(*)...')
#3 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*)...', Array, false)
#4 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
#5 C:\xampp\htdocs\inventaris\app\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
#6 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(934): App\Controllers\Umum->index()
#7 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
#8 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\inventaris\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\inventaris\vendor\codeigniter4\framework\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-02-11 09:22:34 --> Table 'inventaris_smalan.piala' doesn't exist
in SYSTEMPATH\Database\BaseConnection.php on line 645.
 1 SYSTEMPATH\Database\BaseBuilder.php(1686): CodeIgniter\Database\BaseConnection->query('SELECT COUNT(*) AS `numrows`
FROM `piala`', [], false)
 2 SYSTEMPATH\Model.php(605): CodeIgniter\Database\BaseBuilder->countAllResults(true)
 3 APPPATH\Controllers\Umum.php(25): CodeIgniter\Model->countAllResults()
 4 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Umum->index()
 5 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Umum))
 6 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('FCPATH\\index.php')
INFO - 2023-02-11 09:43:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-02-11 09:43:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
